core/src/main/java/hudson/slaves/ChannelPinger.java

```diff
@@ -136,6 +150,7 @@ public void onClosed(Channel channel, IOException cause) {
         });
 
         t.start();
-        LOGGER.fine("Ping thread started for " + channel + " with a " + interval + " minute interval");
+        LOGGER.fine("Ping thread started for " + channel + " with a " +
+            intervalSeconds + " second interval and a " + timeoutSeconds + " second timeout.");
```

Comment:
Without benchmarking, I'd guess overhead for a string concatenation like this is going to be roughly comparable to the overhead of creating/GCing that object array, especially in the context of something that only happens once per slave connection, but I can swap to that.

(My main objection is the ugliness of losing the log-level convenience wrappers + needing to manually make the array... I really wish we'd just ship FormattingLogger as part of guava.)